mod_locations.asciidoc

```diff
@@ -83,8 +84,8 @@ Examples:
 `+https://www.server.com/ocpi/cpo/2.2/locations/?offset=50&amp;limit=100+`
 
 
-If additional parameters: {date_from} and/or {date_to} are provided, only Locations with (`last_updated`) between the given date_from and date_to will be returned.
-If an EVSE is updated, also the 'parent' Location's `last_updated` fields is updated. If a Connector is updated, the EVSE's `last_updated` and the Location's `last_updated` field are updated.
+If additional parameters `{date_from}` and/or `{date_to}` are provided, only Locations with (`last_updated`) between the given `{date_from}` and `{date_to}` will be returned.
+If an EVSE is updated, also the 'parent' Location's `last_updated` field needs to be updated. Analogously, if a Connector is updated, the EVSE's `last_updated` and the Location's `last_updated` fields need to be updated updated.
```

Comment:
I don't like to use words that are bit more complex for most non native English speakers. Could we rephrase to not use: Analogously?

Reply:
I hope _similarly_ is a more common word?

Reply:
similarly is good